packages/cardano-services/test/StakePool/StakePoolHttpService.test.ts

```diff
@@ -473,30 +473,33 @@ describe('StakePoolHttpService', () => {
           expect(responseWithPagination.pageResults).toEqual(responseWithPaginationCached.pageResults);
         });
         it('should paginate rewards response', async () => {
-          const req = { pagination: { limit: 1, startAt: 1 } };
-          const reqWithRewardsPagination = { pagination: { limit: 1, startAt: 1 }, rewardsHistoryLimit: 0 };
-          const responseWithPagination = await provider.queryStakePools(reqWithRewardsPagination);
-          const response = await provider.queryStakePools(req);
-          expect(response.pageResults[0].epochRewards.length).toBeGreaterThanOrEqual(1);
-          expect(responseWithPagination.pageResults[0].epochRewards.length).toEqual(0);
-
-          const responseCached = await provider.queryStakePools(req);
-          expect(response.pageResults).toEqual(responseCached.pageResults);
-          const responsePaginatedCached = await provider.queryStakePools(reqWithRewardsPagination);
-          expect(responseWithPagination.pageResults).toEqual(responsePaginatedCached.pageResults);
+          const reqWithoutRewardsHistory = { pagination: { limit: 1, startAt: 1 } };
+          const reqWithRewardsHistory = { pagination: { limit: 1, startAt: 1 }, rewardsHistoryLimit: 5 };
+          const responseWithRewardsHistory = await provider.queryStakePools(reqWithRewardsHistory);
+          const resNoRewardsHistory = await provider.queryStakePools(reqWithoutRewardsHistory);
+          expect(resNoRewardsHistory.pageResults[0].epochRewards).toHaveLength(3);
+          expect(responseWithRewardsHistory.pageResults[0].epochRewards).toHaveLength(5);
```

Comment:
```suggestion
          expect(resWithoutRewardsHistoryLimit.pageResults[0].epochRewards).toHaveLength(5);
```
This makes the test more expressive. Same feedback applies for the following test too

Reply:
You mean `resWithRewardsHistoryLimit` right? Because this line asserts that we have a rewards history limit of 5.